app/views/collections/_show_document_list_row.html.erb

```diff
@@ -1,10 +1,10 @@
-<% noid = document.noid %>
-<tr id="document_<%= noid %>" class="<%= cycle("", "zebra") %>">
+<% id = document.id %>
```

Comment:
True, though I did miss a bunch of to_param opportunities in this file, so thanks for raising this.